### PR TITLE
Feature: add a checkbox option to keep the PayPal webhooks after disconnect the account

### PR DESCRIPTION
**Pull request by glaubersilva**
Resolves [GIVE-854]

## Description

This PR adds an option to keep the PayPal webhooks config when disconnecting an account, this way we can keep receiving webhook events, including subscription renewals, from the old account. 

## Affects

PayPal donations, especially the recurring donations.

## Visuals

Demo video: https://www.loom.com/share/4bd1ad7a2ecc48d4b875f797425ee30e?sid=258ede42-918b-4342-991a-3f989ed22ff1

## Testing Instructions

1. Connect your PayPal sandbox account;
2. Check if the `give_paypal_commerce_sandbox_webhook_config` is present in your DB
3. Disconnect your PayPal sandbox account;
4. Make sure the `give_paypal_commerce_sandbox_webhook_config` keeps present in your DB

To check if the `give_paypal_commerce_sandbox_webhook_config` is present in your DB, use the following query: 
```
SELECT * FROM `wp_options` WHERE `option_name` = 'give_paypal_commerce_sandbox_webhook_config'
```

## Pre-review Checklist

-   [ ] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [ ] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

[GIVE-854]: https://stellarwp.atlassian.net/browse/GIVE-854?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by JasonTheAdams — APPROVED** [review]
The code looks good to me! I've reached out to @Benunc for a copy check on this. Let's also do a quick DQA on this with @jdghinson before sending it to QA.

File: src/PaymentGateways/PayPalCommerce/ScriptLoader.php [hunk 1]
```diff
@@ -124,6 +125,9 @@ public function loadAdminScripts()
                         'Are you sure you want to disconnect your PayPal account?',
                         'give'
                     ),
+                    'keepWebhooksAfterDisconnect' => esc_html__('Continue to receive events, including subscription renewals, from old account?',
```
Comment (Benunc) [inline src/PaymentGateways/PayPalCommerce/ScriptLoader.php:128]:
```suggestion
                    'keepWebhooksAfterDisconnect' => esc_html__('Continue to receive events, including subscription renewals, from the disconnect account? If you have existing recurring donations with this account, you\'ll want to leave this checked.',
```

File: src/PaymentGateways/PayPalCommerce/ScriptLoader.php [hunk 2]
```diff
@@ -124,6 +125,9 @@ public function loadAdminScripts()
...
+                        'give'),
+                    'keepWebhooksAfterDisconnectLearnMore' => esc_html__('Learn More', 'give') . ' &raquo;',
```
Comment (Benunc) [inline src/PaymentGateways/PayPalCommerce/ScriptLoader.php:130]:
```suggestion
                    'keepWebhooksAfterDisconnectLearnMore' => esc_html__('Learn why this is critically important.', 'give') . ' &raquo;',
```

File: src/PaymentGateways/PayPalCommerce/ScriptLoader.php (hunk 1 above)
Comment (glaubersilva) [reply src/PaymentGateways/PayPalCommerce/ScriptLoader.php]: Resolve in 6a731f5af2093f2d592e242e51c7c12cd4079483

File: src/PaymentGateways/PayPalCommerce/ScriptLoader.php (hunk 2 above)
Comment (glaubersilva) [reply src/PaymentGateways/PayPalCommerce/ScriptLoader.php]: Resolved in 6a731f5af2093f2d592e242e51c7c12cd4079483

**Comment by glaubersilva** [comment]
@jdghinson I recorded a short video demonstrating the visual changes implemented in this PR and would like to get your review on it, here is the link: https://www.loom.com/share/b0dd84e1ba1e4d709e5f1509f6ce256e?sid=50f462e5-62e2-434f-bb85-f9bc7fae7a75

If you need a zip file to install it and check closer, let me know.

**Review by jdghinson — APPROVED** [review]
All good!

**Review by rickalday — APPROVED** [review]
Passed manual QA tests.

**Pull request merged**